rules/aws_cloudtrail_rules/aws_software_discovery.py

```diff
@@ -0,0 +1,39 @@
+from panther_base_helpers import deep_get, aws_rule_context
+
+DISCOVERY_EVENTS = [
+    "ListDocuments",
+    "ListMembers",
+    "DescribeProducts",
+    "DescribeStandards",
+    "DescribeStandardsControls",
+    "DescribeInstanceInformation",
+    "DescribeSecurityGroups",
+    "DescribeSecurityGroupRules",
+    "DescribeSecurityGroupReferences",
+    "DescribeSubnets",
+    "DescribeHub",
+    "ListFirewalls",
+    "ListRuleGroups",
+    "ListFirewallPolicies",
+    "DescribeFirewall",
+    "DescribeFirewallPolicy",
+    "DescribeLoggingConfiguration",
+    "DescribeResourcePolicy",
+    "DescribeRuleGroup",
+]
+
+
+def rule(event):
+    return event.get("eventName") in DISCOVERY_EVENTS
+
+
+def title(event):
+    return (
+        f"User [{deep_get(event, 'userIdentity', 'principalId')}] "
+        f"performed a [{event.get('eventName')}] "
```

Comment:
I know we haven't historically included the Threshold in the title, and I'm wondering if it makes sense to say that the actor has already done `N` number of these items.